sourmash/lca/command_classify.py

```diff
@@ -48,11 +48,15 @@ def classify_signature(query_sig, dblist, threshold):
     # threshold.
 
     tree = {}
-
-    for lca, count in counts.most_common():
-        if count < threshold:
-            break
-
+    lca = []
+
+    if majority_vote:
+        majority = counts.most_common()[0][0]
```

Comment:
error condition to check: what if there are no counts?

Also note, you can pull the first result off of `most_common` with `next(iter(most_common()))`.